apollo-runtime/src/test/java/com/apollographql/android/converter/CacheKeyForFieldTest.java

```diff
@@ -0,0 +1,176 @@
+package com.apollographql.android.converter;
+
+import com.apollographql.android.api.graphql.Field;
+import com.apollographql.android.api.graphql.Operation;
+import com.apollographql.android.api.graphql.util.UnmodifiableMapBuilder;
+
+import junit.framework.Assert;
+
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+
+public class CacheKeyForFieldTest {
+
+  enum Episode {
+    JEDI
+  }
+
+  @Test
+  public void testFieldWithNoArguments() {
+    Field field = Field.forString("hero", "hero", null, false);
+    Operation.Variables variables = new Operation.Variables() {
+      @Nonnull @Override public Map<String, Object> valueMap() {
+        return super.valueMap();
+      }
+    };
+    Assert.assertEquals("hero", field.cacheKey(variables));
+  }
+
+  @Test
+  public void testFieldWithNoArgumentsWithAlias() {
+    Field field = Field.forString("r2", "hero", null, false);
+    Operation.Variables variables = new Operation.Variables() {
+      @Nonnull @Override public Map<String, Object> valueMap() {
+        return super.valueMap();
+      }
+    };
+    Assert.assertEquals("hero", field.cacheKey(variables));
```

Comment:
check out assertJ in some of the other tests.  it offers a nice fluent api:  assertThat(ield.cacheKey(variables)).isEqualTo("hero")   ....not a big deal, just a suggestion